### PR TITLE
Bump versions

### PR DESCRIPTION
**Pull request by Goooler**

### CODE REVIEW

**Comment by Goooler** [comment]
I found two test failures run with Windows, because the regex is not compatible with Windows path

image

**Comment by Goooler** [comment]
Regex is here, paths is splited with `\` in Windows instead of `/`, what you declared should be `"((/|\\\\).*\\.kt):\\d+:\\d+"`

image

**Comment by Goooler** [comment]
When I ran this test again after corrected the `pathPattern`, new case occured, `filePath` miss `C:\` prefix, the file of it could not be specified, the `File(filePath).exists()` is false. 
You may need to test in Windows and fix it.

image

File: build.gradle.kts [hunk 1]
```diff
@@ -12,7 +12,6 @@ plugins {
 repositories {
     mavenCentral()
     google()
-    jcenter() // remove after trove4j resolved https://youtrack.jetbrains.com/issue/KT-44730
```
Comment (jeremymailen) [inline build.gradle.kts:15]: This is still needed, no?

**Comment by jeremymailen** [comment]
True, I've not tried to execute the tests on windows. I will file a bug.

File: build.gradle.kts (hunk 1 above)
Comment (Goooler) [reply build.gradle.kts:15]: Yeah, need not now.

Comment (jeremymailen) [reply build.gradle.kts:15]: Oh good, they finally got it moved
https://youtrack.jetbrains.com/issue/IDEA-261387

**Review by jeremymailen — APPROVED** [review]
ty

**Comment by jeremymailen** [comment]
Fixes #173

**Pull request merged**